src/libs/ajax.js

```diff
@@ -628,6 +638,12 @@ const Billing = signal => ({
       await billing.addProjectUser(projectName, _.difference(newRoles, oldRoles), email)
       return billing.removeProjectUser(projectName, _.difference(oldRoles, newRoles), email)
     }
+  },
+
+  listAzureManagedApplications: async subscriptionId => {
+    const response = await fetchBillingProfileManager(`azure/v1/managedApps?azureSubscriptionId=${subscriptionId}`,
+      _.merge(authOpts(), { signal }))
+    return response.json()
   }
```

Comment:
I believe this more concise equivalent improves readability. Your call:
```suggestion
  listAzureManagedApplications: subscriptionId =>
    fetchBillingProfileManager(`azure/v1/managedApps?azureSubscriptionId=${subscriptionId}`,
      _.merge(authOpts(), { signal })).then(r => r.json())
```

Reply:
I agree this is better, thanks!

Reply:
@dmohs I do still need to have the `return`, and then I get a warning about `async` not having an `await`. Do I just remove the `async`?

Reply:
Actually, I think I'm going to keep the code as-is because it follows the pattern of the other code in Ajax, and I am getting confused then about how some of the error handling would work.

Reply:
When one removes the braces, `return` is implied. For example, the identity function can be defined as:
```javascript
x => x
```
Labeling a function with `async` causes it to return a promise, regardless of whether you `await` within the body. Since the example I gave returns a promise, there is no need for any other decoration.